app/src/org/commcare/fragments/connectId/ConnectIdPhoneVerificationFragmnet.java

```diff
@@ -352,6 +370,9 @@ public void processOldApiError() {
             case MethodVerifyAlternate -> {
                 isBusy = !ApiConnectId.requestVerificationOtpSecondary(requireActivity(), username, password, callback);
             }
+            case MethodUserDeactivate -> {
+                isBusy = false;
```

Comment:
Apart from calling `initiate_deactivation` again here, can we consider calling that API just like we do in other flows? I’m curious because when we discussed this earlier, other flows have the `requestSmsCode` function, which includes sending the OTP.

So, I’m thinking of calling `initiate_deactivation` here with the appropriate method types. By using this approach, we could also manage the resend OTP functionality.

Let me know your thoughts on this.